reference/hoon-errors.md

```diff
@@ -178,3 +187,388 @@ true:
+["|" %give %b %doze [i=//behn/0v1p.sn2s7 t=~]]
+["|" %give %b %wake [i=/g/use/spider/~zod/thread/~.dojo_0v6.210tt.1sme1.ev3qm.qgv2e.a754u/wait/~2020.1.14..19.01.26..7556 t=~[/d //term/1]]]
+["||" %give %g [%unto %fact] [i=/g/use/dojo/~zod/out/~zod/spider/drum/wool t=~[/d //term/1]]]
+["|||" %give %g [%unto %fact] [i=/g/use/hood/~zod/out/~zod/dojo/drum/phat/~zod/dojo t=~[/d //term/1]]]
+["||||" %give %g [%unto %fact] [i=/d t=~[//term/1]]]
+["|||||" %give %d %blit [=//term/1 t=~]]
+["||" %give %g [%unto %kick] [i=/g/use/dojo/~zod/out/~zod/spider/drum/wool t=~[/d //term/1]]]
+~s1..0007
+```
+This gives us a stack trace that is a list of `move`s and some
+associated metadata. Some of the `move`s are a bit of a distraction from what's
+going on overall such as acknowledgements that a `poke` was received
+(`%poke-ack`s), so we've omitted them for clarity. Furthermore, two `move`s (the
+`%blit` `card`s) is not printed even in verbose mode because it occurs so
+frequently, but there are only two of them here and so we have added it in.
+
+The main process that is occurring here is a sequence of `%pass` `move`s initiated
+by pressing Enter in the terminal that goes on to be handled by Dill, then Gall,
+and finally Behn. After the timer has elapsed, a sequence of `%give` `move`s is
+begun by Behn, which then passes through Gall and ultimately ends up back at the
+terminal. Any `move`s besides `%pass` in the first segment of the stack trace is a
+secondary process utilized for book-keeping, spawning processes, interpreting
+commands, etc. All of this will be explained in detail below.
+
+
+It is important to note that this stack trace should be thought of
+as being from the "point of view" of the kernel - each line represents the
+kernel taking in a message from one source and passing it along to its
+destination. It is then processed at that destination (which could be a vane or
+an app), and the return of that process is sent back to Arvo in the form of
+another `move` to perform and the loop begins again. Thus this stack trace does
+not display information about what is going on inside of the vane or app such as
+private function calls, only what the kernel itself sees.
+
+### Interpreting the stack trace
+
+In this section we will go over the stack trace line-by-line, explaining how the
+stack trace is printed, what each line means (including some things not found in
+the stack trace), and a particular focus on what code is being activated in the
+first few lines that should equip you well enough to unravel the rest of the
+stack trace in as much detail as you desire.
+
+
+#### The call
+
+Let's put the first part of the stack trace into a table to make reading a little easier.
+
+| Length | move    | vane(s)   |                                                                                                     action | duct                                                                            |
```

Comment:
I would associate the `%pass` wires with the either the `%pass` or the duct, probably the former.  Seems like a more natural place than with the action.  (Maybe this should be changed in the `|verb` output too, but out of scope for this)